frontend/src/libs/components/profile-settings-sidebar/styles.module.scss

```diff
@@ -0,0 +1,59 @@
+.container {
+  max-width: 349px;
+  min-height: 100vh;
+  background: linear-gradient(180deg, rgb(229 240 255 / 0%) 0%, #e5f0ff 100%);
```

Comment:
You should use constants frrom variables with only hex, not rgb

Reply:
Well, that' the tricky part, in some cases, an element in Figma would have a hex that we do not have in our variables

Reply:
In this case you can add new one, all variables are there because someone was needed them)

Reply:
That makes sense, just did not really want to mess with global variables until I get some sort of approval